Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Builder
+FROM golang:alpine as builder
+
+WORKDIR /build
+
+# Build deps
+RUN apk add --no-cache bash perl git
+
+# Copy source
```

Comment:
What happened to the `COPY go.mod go.sum .` and `RUN go mod download` section? Did you remove it intentionally?